Library/Homebrew/compat/early/download_strategy.rb

```diff
@@ -17,7 +17,7 @@ def _fetch(*args, **options)
       rescue ArgumentError => e
         raise unless e.message.include?("timeout")
 
-        odisabled "`def _fetch` in a subclass of `CurlDownloadStrategy`"
+        odeprecated "`def _fetch` in a subclass of `CurlDownloadStrategy`"
```

Comment:
I suggest adding a comment here that says when this was `odeprecated`.

If you think that's useful, I could go through the existing `odeprecated`s and comment accordingly.